api/server/router/local/image.go

```diff
@@ -269,7 +279,7 @@ func (s *router) postBuild(ctx context.Context, w http.ResponseWriter, r *http.R
 	var (
 		authConfigs        = map[string]cliconfig.AuthConfig{}
 		authConfigsEncoded = r.Header.Get("X-Registry-Config")
-		buildConfig        = builder.NewBuildConfig()
+		buildConfig        = new(dockerfile.Config)
```

Comment:
same here, it should be defined way below as `buildConfig := &dockerfile.Config{}`. I'm okay with `new`, but seems like in other code we use `&`.